client/homebrew/pages/homePage/welcome_msg.md

```diff
@@ -48,57 +48,62 @@ If you want to save ink or have a monochrome printer, add the **PRINT → {{fas,
 
 \column
 
-## New in V3.0.0
-We've implemented an extended Markdown-like syntax for block and span elements, plus a few other changes, eliminating the need for HTML tags like `div` and `span` in most cases. No raw HTML tags should be needed in a brew (*but can still be used if you insist*).
+## V3 vs Legacy
+The Homebrewery has two renderers: Legacy and V3. The V3 renderer is recommended and will receive updates, while the Legacy renderer will not. Legacy mode is still available for older brews and veteran users, but won't be updated.
+:
+V3 uses curly brackets instead of HTML tags, though HTML tags can still be used. Converting between Legacy and V3 may require tweaking the document. However, all legacy brews will still render normally. Both renderers can be accessed via the {{fa,fa-info-circle}} **Properties** button on your brew at any time.

```

Comment:
```suggestion
The Homebrewery has two renderers: Legacy and V3. The V3 renderer is recommended for all users because it is more powerful, more customizable, and continues to receive new feature updates while Legacy does not. However Legacy mode will remain available for older brews and veteran users.
	
At any time, any individual brew can be changed to your renderer of choice via the {{fa,fa-info-circle}} **Properties** tab on your brew. However, converting between Legacy and V3 may require heavily tweaking the document; while both renderers can use raw HTML, V3 prefers a streamlined curly bracket syntax that avoids the complex HTML structures required by Legacy.
```